courses/machine_learning/deepdive2/structured/solutions/3b_bqml_linear_transform_babyweight.ipynb

```diff
@@ -0,0 +1,1162 @@
+{
```

Comment:
<p>Prefix this cell by the following comment:</p><p> </p><p>"Let's select the <code>mean_square_error</code> from the evaluation table we just computed and square it to obtain the rme"</p><p> </p><p>Alternatively, I'd remove this cell altogether, since it seems a bit much to redo the whole evaluation just to take the square of the mse. </p><p> </p><p>A even better option would be to use the <code>--destination_table</code>&nbsp;argument of %%bigquery to store the evaluation results in BQ in the previous</p><p>cell and then retrieve in this cell directly from this saved table. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/669/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Done</p>